web/src/pages/vendor-program/VendorProgramSubmissionViewPage.vue

```diff
@@ -86,19 +89,19 @@ watch(
     if (newVal && newVal.id) setBreadcrumbs()
   }
 )
+
 setBreadcrumbs()
```

Comment:
It might be worth upgrading this breadcrumb implementation to the one used in wrap https://github.com/icefoganalytics/wrap/blob/main/web/src/use/use-breadcrumbs.

That way you can pass a computed property.
e.g.
```ts
const breadcrumbs = computed(() => ...)
useBreadcrumbs(breadcrumbs)
```
And it'll be a bit cleaner.